avocado/utils/configure_network.py

```diff
@@ -43,6 +43,36 @@ class For network interface
     def __init__(self, interface):  # pylint: disable=W0231
         self._interface = interface
```

Comment:
what about :

```python3

def __init__(self, if_name):
   self.name = if_name
```

So you don't need the `def get_name()` at the end of this class, right? IIUC, the name should not be private.